programs/vote/src/vote_state/mod.rs

```diff
@@ -84,13 +84,16 @@ impl Lockout {
         (INITIAL_LOCKOUT as u64).pow(self.confirmation_count)
     }
 
-    // The slot height at which this vote expires (cannot vote for any slot
-    // less than this)
-    pub fn expiration_slot(&self) -> Slot {
+    // The last slot at which a vote is still locked out. Validators should not
+    // vote on a slot in another fork which is less than or equal to this slot
+    // to avoid having their stake slashed.
+    pub fn last_locked_out_slot(&self) -> Slot {
         self.slot + self.lockout()
     }
-    pub fn is_expired(&self, slot: Slot) -> bool {
-        self.expiration_slot() < slot
+
+    // Check if a lockout has expired before checked slot.
+    pub fn has_expired_by_slot(&self, slot: Slot) -> bool {
```

Comment:
how about `is_locked_out_at_slot` and flipping this to `>=`

Reply:
switched